DevCycle.SDK.Server.Common/API/DVCBaseApiClient.cs

```diff
@@ -31,7 +31,7 @@ public virtual async Task<RestResponse> SendRequestAsync(object json, string url
 
             if (shouldRetry)
             {
-                return await ClientPolicy.GetInstance().RetryPolicyWithTimeout.ExecuteAsync(() => restClient.ExecuteAsync(request));
+                return await ClientPolicy.GetInstance().CloudClientRetryPolicyWithTimeout.ExecuteAsync(() => restClient.ExecuteAsync(request));
```

Comment:
didnt you say this one was shared between cloud and local? Maybe the retry policy it uses here should just be named "Standard" or something, and the other one could be "ConfigFetch"

Reply:
yup, you're right!